src/guide/jbs-jdk-bug-system.md

```diff
@@ -211,15 +219,15 @@ Once you are made, or you make yourself, the assignee of an Issue you take on th
 [JDK-8212207](https://bugs.openjdk.org/browse/JDK-8212207),
 [JDK-6984895](https://bugs.openjdk.org/browse/JDK-6984895),
 [JDK-8287982](https://bugs.openjdk.org/browse/JDK-8287982)
-), this is especially helpful if you ultimately move the Issue to a different area as you have found that the problem lies elsewhere, or is closed as [Will Not Fix]{.jbs-field}.  Your updates then provide a resource to others to better understand what has been done or the code. See [The Importance of Writing Stuff Down](https://stuartmarks.wordpress.com/2023/02/22/the-importance-of-writing-stuff-down) for a good explanation as to why it is also important to write an evaluation when something is not fixed.
+), this is especially helpful if you ultimately move the Issue to a different area as you have found that the problem lies elsewhere, or is closed as [Will Not Fix]{.jbs-field}.  Your updates then provide a resource to others to better understand what has been done or the code. See [The Importance of Writing Stuff Down](https://stuartmarks.wordpress.com/2023/02/22/the-importance-of-writing-stuff-down) for a good explanation as to why it is important.
 
 Some additional fields should be filled in as you get a better understanding of the issue:
 - for a regression, if you identify the fix that caused it add a link to that issue (and add a [regression_]{.jbs-field}*(ID)* label) and set the _Introduced in Release_ field.
 - The [Description]{.jbs-field} usually explains what went wrong and how the failure was found, then there's some investigation and eventually the root cause is found. At this point the [Summary]{.jbs-field} should be updated to correctly describe the bug. The [Description]{.jbs-field} however should remain a description of how the failure was found.
 
 ## Resolving an issue
 
-Once the work on an issue has been completed this should be indicated in JBS by moving the issue to a "closed" state.  There are two "closed" states: [Resolved]{.jbs-field} and [Closed]{.jbs-field} which can be used interchangeably except in the case of [Fixed]{.jbs-field}, or when flagged as  [Incomplete]{.jbs-field} (See Triaging).
+Once the work on an issue has been completed the issue should be in a "closed" state.  There are two "closed" states: [Resolved]{.jbs-field} and [Closed]{.jbs-field} which can be used interchangeably except in the case of [Fixed]{.jbs-field}, or when flagged as  [Incomplete]{.jbs-field} (See Triaging).
```

Comment:
There is also a discrepancy in terms of JBS operation when it comes to closing as a duplicate. If you use "Close" then you get the UI that allows you to set the Resolution as "Duplicate" and link to the issue which is duplicated. However, if you use Resolve then the ability to link in the UI is not there.

Reply:
That is a good point and leads to bugs being closed as duplicates without a duplicate link - I have a "filter" to look for these.

in the duplicate section of the following table it says:

Remember to add a [duplicates]{.jbs-field} link - not a [relates to]{.jbs-field} link</td>

Let me make that more explicit with the text you have above

Reply:
done